src/gt4py/next/otf/binding/nanobind.py

```diff
@@ -28,6 +29,7 @@ class Expr(eve.Node):
 
 class DimensionType(Expr):
```

Comment:
```suggestion
class DimensionSpec(Expr):
```
Unrelated, but feels to me like this would blend more with the rest of the file.